Notebooks/QC/formats.ipynb

```diff
@@ -70,7 +68,7 @@
     "   \n",
     "From this we can calculate the [Phred quality score](https://en.wikipedia.org/wiki/Phred_quality_score) as:\n",
     "   \n",
-    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; P = 10<sup>-Q/10</sup>   \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Q = 10<sup>-Q/10</sup>   \n",
```

Comment:
No. The formula was correct. What we need to clarify is that the Phred quality score is `Q`, whereas `P` is the base calling error probablity.